test_setup_game.py

```diff
@@ -57,6 +60,14 @@ def test_create_players(self):
             self.assertEqual(
                 players.get_players()[0].get_name(), expected_players.get_players()[0].get_name())
 
+        with self.subTest(
+                'should set the players to computer vs human if computer is selected as opponent'):
```

Comment:
small feedback -- should this be something like "should set players to super computer vs human if super computer is selected" instead? since you're differentiating the two computer options

Reply:
True! That's alot clearer!